Monika After Story/game/script-holidays.rpy

```diff
@@ -6902,19 +6902,47 @@ init -1 python:
     def mas_isMonikaBirthday(_date=None):
         """
         checks if the given date is monikas birthday
-        Comparison is done solely with month and day
-
+        converts date to datetime and runs thru mas_isMonikaBirthday_dt
```

Comment:
undo this change, the description here should be explicit on how the comparison is done, and mentioning its a passthrough is only if the function documentation is too large to copy.